ci-regression.sh

```diff
@@ -17,31 +17,36 @@ if [ -d "${TEST_HARNESS_MASTER_DIR}" ]; then
   fi
 fi
 
+mkdir -p "${PWD}/test-harness-out"
+
 echo "Copying ${PWD} to ${TEST_HARNESS_MASTER_DIR}"
 cp -p -r "${PWD}" "${TEST_HARNESS_MASTER_DIR}"
 
 pushd "${TEST_HARNESS_MASTER_DIR}"
```

Comment:
If this fails, the hard reset below will ruin lives.

Reply:
Agreed, and it would add insult to injury considering fundies runs these tests locally because he can.